### PR TITLE
Added new methods for checking Value and Text in tests using waitUntil

### LINKED ISSUE #1448: always use waitForContent when testing for presense of text
- test content always with `element.waitForContent("expected content")`. This will keep checking for presense of expected content repeatedly for up to 15 seconds and then throw. That way, you avoid "testing too early" which is the cause of 95% of flakiness of UI tests. This is what we do on browser extension.

_Originally posted by @tomholub in https://github.com/FlowCrypt/flowcrypt-ios/issues/1056#issuecomment-974681036_

### PR DESCRIPTION
**Pull request by fcvakintos**
This PR contains 2 new methods for checking text and value in elements, The methods will wait 15 seconds until the text or value will present in element, after will return the error - "expected value/text to be different after 15s". When the text is appears before 15s, the methods will complete.
close #1448 

----------------------------------

**Tests** 
- Tests added or updated

--------------------------------

### To be filled by reviewers

I have reviewed that this PR... _(tick whichever items you personally focused on during this review)_:
- [ ] addresses the issue it closes (if any)
- [ ] code is readable and understandable
- [ ] is accompanied with tests, or tests are not needed
- [ ] is free of vulnerabilities
- [ ] is documented clearly and usefully, or doesn't need documentation

### CODE REVIEW

File: appium/tests/helpers/ElementHelper.ts [hunk 1]
```diff
@@ -60,6 +60,29 @@ class ElementHelper {
...
+  }
+
+  //wait for value in element during 15 seconds (if the value doesn't appear during 15s, it will show the error)
+  static waitForValue = async (element: WebdriverIO.Element, value: string, timeout: number = DEFAULT_TIMEOUT) => {
+    await this.waitElementVisible(element);
+    await element.waitUntil(async  function () {
+      return (await element.getValue() === value)
+    }, {
+      timeout: timeout,
+      timeoutMsg: 'expected value to be different after 15s'
```
Comment (tomholub) [inline appium/tests/helpers/ElementHelper.ts:82]: Do `expected value within ${timeout}s to be "${value}" but got last value "${lastValue}"`

Similarly above

File: appium/tests/screenobjects/new-message.screen.ts [hunk 2]
```diff
@@ -216,8 +217,12 @@ class NewMessageScreen extends BaseScreen {
...
     const recipientCell = await $(`~aid-${type}-${order}-label`);
     await ElementHelper.waitElementVisible(recipientCell);
-    const name = await recipientCell.getValue();
-    expect(name).toEqual(`  ${recipient}  `);
+    await recipientCell.waitUntil(async  function () {
+      return (await recipientCell.getValue() === `  ${recipient}  `)
+    }, {
+      timeout: 15,
+      timeoutMsg: 'expected text to be different after 15s'
+    });
```
Comment (tomholub) [inline appium/tests/screenobjects/new-message.screen.ts:225]: Should use the new method here

Comment (fcvakintos) [reply appium/tests/screenobjects/new-message.screen.ts:225]: do you mean another method?

Comment (tomholub) [reply appium/tests/screenobjects/new-message.screen.ts:225]: You've just implemented a `waitForValue` method. Why not use it here?

**Review by tomholub — APPROVED** [review]

**Pull request merged**